tutoroars/templates/oars/apps/scripts/oars/dbt.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+python3 -m venv virtualenv
+. virtualenv/bin/activate
+
+echo "Installing dbt packages..."
+pip install pip install {{ DBT_PACKAGES }}
```

Comment:
Is this (and the equivalent line in the init script) supposed to be just `pip install {{ DBT_PACKAGES }}`?